src/LightstonePlatform.Products/LightstonePlatform.Products.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Why this bump?

Reply:
reverted as this was a change needed when referencing the lightstone assemblies which was removed